src/converters.js

```diff
@@ -885,6 +885,8 @@ function injectViewList( modelItem, injectedItem, mapper, removePosition ) {
 		}
 	}
 
+	insertPosition = positionAfterUiElements( insertPosition );
```

Comment:
Can't it simply be a position at the end of `<li>`? I don't think we suppose to have anything else after a nested `<ul>`?